### ISSUE TITLE
[BUG]Accessibility issue with tgl-tab class

### ISSUE THREAD

**Issue by pmirowski** [wet-boew/wet-boew#10187]
The issue is similar to the accessibility problems with tab panels, where the class "tgl-tab" has missing aria-labels and the hidden content for the accordion causes errors. Please look into https://playwright.dev/docs/accessibility-testing 

"failureSummary": "Fix any of the following:
   Element has focusable descendants",
         "html": "<div class=\"tgl-tab\" role=\"tab\" aria-selected=\"false\" tabindex=\"-1\" aria-posinset=\"20\" aria-setsize=\"21\">",
         "impact": "serious",
         "none": Array [],
         "target": Array [
           "div[aria-posinset=\"20\"]",
         ],
       },
       Object {
         "all": Array [],
         "any": Array [
           Object {
             "data": null,
             "id": "no-focusable-content",
             "impact": "serious",
             "message": "Element has focusable descendants",
             "relatedNodes": Array [
               Object {
                 "html": "<summary class=\"wb-toggle tgl-tab wb-init wb-toggle-inited\" data-toggle=\"{&quot;parent&quot;: &quot;.accordion&quot;, &quot;group&quot;: &quot;.acc-group&quot;}\" id=\"wb-auto-45\">Voyage</summary>",
                 "target": Array [
                   "#wb-auto-45",
                 ],
               },
             ],
           },
         ],

   Object {
     "description": "Ensure interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies",
     "help": "Interactive controls must not be nested",
     "helpUrl": "https://dequeuniversity.com/rules/axe/4.11/nested-interactive?application=playwright",
     "id": "nested-interactive",
     "impact": "serious",
     "nodes": Array [
       Object {
         "all": Array [],
         "any": Array [
           Object {
             "data": null,
             "id": "no-focusable-content",
             "impact": "serious",
             "message": "Element has focusable descendants",
             "relatedNodes": Array [
               Object {
                 "html": "<summary class=\"wb-toggle tgl-tab wb-init wb-toggle-inited\" data-toggle=\"{&quot;parent&quot;: &quot;.accordion&quot;, &quot;group&quot;: &quot;.acc-group&quot;}\" id=\"wb-auto-3\" tabindex=\"0\"><span id=\"cegpta\" class=\"h3\">Conseil exécutif de gestion du Programme du travail (CEGPT)</span></summary>",
                 "target": Array [
                   "#wb-auto-3",
                 ],
               },
             ],
           },
         ],
         "failureSummary": "Fix any of the following:
   Element has focusable descendants",
         "html": "<div class=\"tgl-tab\" role=\"tab\" aria-selected=\"false\" tabindex=\"-1\" aria-posinset=\"1\" aria-setsize=\"8\">",
         "impact": "serious",
         "none": Array [],
         "target": Array [
           ".tgl-tab[role=\"tab\"][aria-posinset=\"1\"]",
         ],
       },

Please look into it. 

example:
https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html
<img width="554" height="382" alt="Image" src="https://github.com/user-attachments/assets/2477246d-af09-4038-bd22-8a4b6e9b3e19" />

**Comment by pmirowski** [comment]
Tab index also seems to be causing an issue

<img width="1317" height="502" alt="Image" src="https://github.com/user-attachments/assets/65bde051-b1e2-4366-bf0b-c2f7ed63b17a" />

<img width="656" height="754" alt="Image" src="https://github.com/user-attachments/assets/43191195-8b43-42d0-aa91-221e57e8ff44" />
